src/QsCompiler/TestProjects/test11/test11.csproj

```diff
@@ -1,14 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.27.253010">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <PlatformTarget>x64</PlatformTarget>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Standard" Version="0.6.1905.301" />
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.6.1905.301" />
-  </ItemGroup>
```

Comment:
Why are these projects getting changed from the .NET SDK to the Quantum SDK and the packages removed as part of upgrading Newtonsoft?

Reply:
Removing unnecessary dependencies in the tests seems like a positive.

Reply:
This particular fragment that is being deleted refers to the (very) outdated QDK (that refers the vulnerable external dependencies). I believe this was the way the quantum programs were written before the `Microsoft.Quantum.Sdk` was developed. I upgraded the project according to the [migration guide](https://learn.microsoft.com/en-us/azure/quantum/install-update-qdk).

Reply:
Right. But I thought some of the value of these tests was to ensure old projects could still open in the latest builds. Maybe I'm mistaken.

Reply:
I'm inclined to agree.

But if we leave the test11 as is (revert in this PR the test11 change) then I believe one particular Component Governance issue will continue being reported (the one that you can navigate to from the DevOps item 48477, then search for "test11" in the Component Governance issue).